src/Model/FileExplorer.rei

```diff
@@ -4,16 +4,20 @@ type t = {
   tree: option(FsTreeNode.t),
   isOpen: bool,
   scrollOffset: [ | `Start(float) | `Middle(float)],
-  focus: option(string) // path
+  active: option(string), // path
+  focus: option(string) // node id
```

Comment:
Note to self: path, not id

Reply:
That would be one nice thing about using a path library like `Path`/`fp` - we could have a type for it